packages/web/pages/_app.tsx

```diff
@@ -138,11 +139,19 @@ function MyApp({ Component, pageProps }: AppProps) {
           transition={Bounce}
         />
         <MainLayout menus={menus}>
-          <Component {...pageProps} />
+          {Component && <Component {...pageProps} />}
         </MainLayout>
       </StoreProvider>
     </GetKeplrProvider>
   );
 }
 
-export default MyApp;
+export default withLDProvider({
+  clientSideID: process.env.NEXT_PUBLIC_LAUNCH_DARKLY_CLIENT_SIDE_ID || "",
+  user: {
+    anonymous: true,
+  },
+  options: {
+    bootstrap: "localStorage",
+  },
+})(MyApp as ComponentType<{}>);
```

Comment:
Are we sure this won't disable SSR and automatic route splitting?

Reply:
I just reviewed their code. This will just wrap the component in a React Provider